kentikapi/cloud_exports_integration_test.go

```diff
@@ -523,13 +523,12 @@ func TestClient_CreateCloudExport(t *testing.T) {
 				assert.NoError(t, err)
 			}
 
-			if assert.Equal(t, 1, len(server.requests.createCERequests), "invalid number of requests") {
+			if tt.request != nil && assert.Equal(t, 1, len(server.requests.createCERequests), "invalid number of requests") {
```

Comment:
```suggestion
			if tt.expectedRequest != nil && assert.Equal(t, 1, len(server.requests.createCERequests), "invalid number of requests") {
```
Little more readable. If there is no expected request, do not verify it.
Also, the name of the test "nil request, status InvalidArgument received" should be changed to e.g. "nil request", because no request is performed in that case. Also, `tt.response` should be removed there, because it is not needed and we want to keep code lean.

Reply:
Done